### PR TITLE
Completes task 6

### PR DESCRIPTION
**Pull request by ollycrossley**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -37,67 +37,104 @@ describe('Endpoint Tests', () => {
...
+                    expect(body.msg).toBe('invalid id')
+                })
+            });
         });
-        test('GET 400   | Return 400 and sends an appropriate error message when given an invalid id', () => {
-            return request(app).get('/api/articles/three').expect(400).then(({body}) => {
-                expect(body.msg).toBe('invalid id')
-            })
+        describe('/api/articles/:article_id/comments', () => {
+            test('GET 200   | Return 200 and array with correct comment objects', () => {
```
Comment (xViolaine) [inline __tests__/app.test.js:109]: We would still expect a test with a 200 if the article exists, when there are no comments

File: models/articles-models.js [hunk 2]
```diff
@@ -21,4 +21,15 @@ exports.selectArticleById = (param_id) => {
...
         return rows[0];
     })
+}
+
+exports.selectArticleComments = (param_id) => {
+    return db.query(`
+    SELECT * FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC
+    `, [param_id]).then(({rows}) => {
```
Comment (xViolaine) [inline models/articles-models.js:30]: See comment re: testing - if the ID is valid and the article exists but there are no comments for the article, we just want to return an empty array

**Review by xViolaine — CHANGES_REQUESTED** [review]
Hey there! Added some comments for you to consider, please resubmit :)

**Review by MRombi — APPROVED** [review]
Good to merge 🙂

**Pull request merged**